dts/bindings/clock/st,stm32wb-rcc.yaml

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2021, Linaro ltd
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  STM32WB Reset and Clock controller node.
+  For more description confere st,stm32-rcc.yaml
+
+compatible: "st,stm32wb-rcc"
+
+include:
+  - name: st,stm32-rcc.yaml
+    property-blocklist:
+      - ahb-prescaler
+
+properties:
+    cpu1-prescaler:
+      type: int
+      required: false
+      enum:
+        - 1
```

Comment:
Same question here about whether this property is needed yet.

Reply:
Same answer as this is the exact same `ahb-prescaler` behind the scene.